docs/source/api_ref_modules.rst

```diff
@@ -82,11 +82,11 @@ Loss
    :toctree: generated/
    :nosignatures:
 
-   loss.PPOLoss
```

Comment:
thoughts on moving these as a subsection under RLHF? @joecummings 
not married to either

Reply:
Subsection makes sense to me

Reply:
@ChinoUkaegbu thanks for updating these!
Could you please move the Loss section to under the RLHF section at the bottom of the file?
You'll also need to change the heading from 
```
Loss
------------------
```
to 
```
Losses
^^^^^
```

See [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections) for more details

Reply:
Done!